fix: types

BunsDev opened the pull request.

BunsDev committed:
chore: migrate to pnpm and fix motion types in input component

BunsDev committed:
chore: migrate to pnpm and fix TypeScript types in motion animations

Reche13 commented on src/app/docs/layout.tsx:
```diff
@@ -1,3 +1,4 @@
+"use client";
```
Just wondering, is there a reason for "use client" here?

BunsDev replied:
just best practice in the latest next versions

Reche13 commented on package.json:
```diff
@@ -18,7 +18,8 @@
     "clsx": "^2.1.1",
     "lucide-react": "^0.510.0",
-    "motion": "^12.11.3",
+    "framer-motion": "^12.23.11",
```
Since motion was already included, is there a reason for adding Framer Motion here?

BunsDev replied:
so long as it works, it's all good

Reche13 commented on src/components/site/tabs.tsx:
```diff
@@ -38,17 +38,17 @@ const TabsList = React.forwardRef<
+  const triggerRef = useRef<HTMLButtonElement>(null);
+  const [isActive, setIsActive] = useState(false);
+  const tabsId = useContext(TabsContext);
+  console.log({ ref })
```
console.log left in. remove?

BunsDev replied:
yeah, remove, apologies

vercel[bot] committed:
Fix React Server Components CVE vulnerabilities

Updated dependencies to fix Next.js and React CVE vulnerabilities.

The fix-react2shell-next tool automatically updated the following packages to their secure versions:
- next
- react-server-dom-webpack
- react-server-dom-parcel  
- react-server-dom-turbopack

All package.json files have been scanned and vulnerable versions have been patched to the correct fixed versions based on the official React advisory.

Co-authored-by: Vercel <vercel[bot]@users.noreply.github.com>

BunsDev committed:
Merge pull request #1 from BunsDev/vercel/react-server-components-cve-vu-4af8uu

Fix React Server Components CVE vulnerabilities
